zhaquirks/xiaomi/aqara/vibration_aq1.py

```diff
@@ -69,6 +69,8 @@
 class VibrationAQ1(XiaomiQuickInitDevice):
     """Xiaomi aqara smart motion sensor device."""
 
+    quirk_id = "xiaomi_aqara_vibration_aq1"
```

Comment:
Can you add it here https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/quirk_ids.py and then reference the quirk ID?

We'll also need to do a quirks bump and change in HA.

Reply:
Yes, thanks.